Packs/RansomwareResponsePackForICSSystemsUsingCortexXSOARAndXDR/IncidentFields/incidentfield-Alert_Type_ID.json

```diff
@@ -0,0 +1,26 @@
+{
+ "associatedToAll": true,
```

Comment:
Ideally, the incident fields should be limited to just the relevant incident fields.
I suggest changing this to just the new incident field that should be added.

Reply:
I had to use new custom fields since this is a different category of incident namely operational tech related impacting ICS systems.